CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape.uc

```diff
@@ -175,29 +192,57 @@ simulated protected function BuildActionInfoTop()
 	ActionInfoTopContainer.SetPosition(0, 150);
 	ActionInfoTopContainer.SetSize(960, 195);
 
-	ActionInfoBG = Spawn(class'UIBGBox', ActionInfoTopContainer);
-	ActionInfoBG.bAnimateOnInit = false;
-	ActionInfoBG.InitBG('ActionInfoBG');
-	ActionInfoBG.SetAlpha(60);
-	ActionInfoBG.SetPosition(-UI_INFO_BOX_MARGIN, -UI_INFO_BOX_MARGIN);
-	ActionInfoBG.SetSize(ActionInfoTopContainer.Width + UI_INFO_BOX_MARGIN * 2, ActionInfoTopContainer.Height + UI_INFO_BOX_MARGIN * 2);
+	BuildActionImage();
+	BuildActionBrief();
+}
 
-	ActionDisplayNameBG = Spawn(class'UIImage', ActionInfoTopContainer);
+simulated protected function BuildActionImage()
+{
+	/*ActionImageBorder = Spawn(class'UIImage', ActionInfoTopContainer);
+	ActionImageBorder.bAnimateOnInit = false;
+	ActionImageBorder.InitImage('ActionImageBorder', "img:///UILibrary_CovertInfiltration.Ops_Border_Full");
+	ActionImageBorder.SetPosition(-10, -2);
+	ActionImageBorder.SetSize(320, 172);*/
```

Comment:
I think this can be removed?

Reply:
The image will probably look better with border but i'm too tired to size and position it. But I left it commented out so that when later the design is done there is less code to write